components/script/dom/offscreencanvas.rs

```
@@ -162,6 +247,12 @@ impl OffscreenCanvasMethods<crate::DomTypeHolder> for OffscreenCanvas {
         _options: HandleValue,
         can_gc: CanGc,
     ) -> Fallible<Option<RootedOffscreenRenderingContext>> {
+        // Always throw an InvalidState exception when the context mode is
+        // detached (See table in the spec).
```

I would add spec text `Throw an ["InvalidStateError"](https://webidl.spec.whatwg.org/#invalidstateerror) [DOMException](https://webidl.spec.whatwg.org/#dfn-DOMException).`